tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -577,10 +577,7 @@
       runtime-id: Guid_10
     },
     Metrics: {
-      process_id: 0,
-      _dd.top_level: 1.0,
-      _dd.tracer_kr: 1.0,
-      _sampling_priority_v1: 1.0
```

Comment:
You shouldn't lose all of these 😕 This seems to stem from the fact you have accidentally added a new span